build.cake

```diff
@@ -117,12 +117,11 @@ Task("GetReleaseNotes")
             Information("Adding metadata for " + addinSpec.Name);
             string fileContent = FileReadText(addinSpec.ReleaseNotesFilePath);
             DeleteFile(addinSpec.ReleaseNotesFilePath);
-            var title = addinSpec.Categories.Contains("Core") ? addinSpec.Name : "Release Notes";
             var frontMatter =
                 new List<string>
                 {
                     "---",
-                    "Title: " + title,
+                    "Title: " + addinSpec.Name + " Release Notes",
```

Comment:
Typo: **r**elease **n**otes should be lowercase?

Reply:
Currently titles are always with upper-cased words. For the sake of consistency it should be the same here.